databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -22,11 +20,6 @@
 from databuilder.publisher.neo4j_preprocessor import NoopRelationPreprocessor
 
 
-# Setting field_size_limit to solve the error below
-# _csv.Error: field larger than field limit (131072)
-# https://stackoverflow.com/a/54517228/5972935
-csv.field_size_limit(int(ctypes.c_ulong(-1).value // 2))
```

Comment:
oh, we no longer set this line?

Reply:
oh, nvm, read your latter comment

Reply:
Right. `csv` module is no longer used in this module therefore taking it out. 

I'm wondering if `csv.field_size_limit()` is a global setting. If that's the case, it could impact other modules that uses `csv` though.  I will put it back to make sure it's backward compatible. It could be better if it's moved into somewhere that is "global" later.

Reply:
sounds good, let's keep it here for now and we could remove it later. cc @jinhyukchang